features/proxy_config.feature

```diff
@@ -61,12 +55,12 @@ Feature: Proxy configuration
          \*   sudo ua config set apt_http_proxy=<value>
          \*   sudo ua config unset apt_http_proxy
          \*/
-        Acquire::http::Proxy ".*";
-        Acquire::https::Proxy ".*";
+        Acquire::http::Proxy::esm.ubuntu.com ".*";
+        Acquire::https::Proxy::esm.ubuntu.com ".*";
         """
         When I run `apt update` with sudo
         And I run `cat /var/log/squid/access.log` `with sudo` on the `proxy` machine
-        Then stdout matches regexp:
+        Then stdout does not match regexp:
```

Comment:
Let's also check on the `squid` logs if we have `CONNECT esm.ubuntu.com:443` entries on it

Reply:
Makes sense, let me add it.